Sources/JWTKit/JWTError.swift

```diff
@@ -4,7 +4,7 @@ public enum JWTError: Error, CustomStringConvertible, LocalizedError {
     case claimVerificationFailure(name: String, reason: String)
     case signingAlgorithmFailure(Error)
     case malformedToken
-    case signatureVerifictionFailed
+    case signatureVerificationFailed
```

Comment:
This doesn't have to be a semver-major change. Just add this:
```swift
extension JWTError {
    @available(*, deprecated, renamed: "signatureVerificationFailed")
    public var signatureVerifictionFailed; JWTError { .signatureVerificationFailed }
}
```

Reply:
But @davdroman if you want to update it we can add it to v4